packages/flutter/test/material/circle_avatar_test.dart

```diff
@@ -3,24 +3,79 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  testWidgets('CircleAvatar test', (WidgetTester tester) async {
+  testWidgets('CircleAvatar with background color', (WidgetTester tester) async {
+    final Color backgroundColor = Colors.blue.shade400;
     await tester.pumpWidget(
       new Center(
         child: new CircleAvatar(
-          backgroundColor: Colors.blue[400],
+          backgroundColor: backgroundColor,
           radius: 50.0,
-          child: new Text('Z')
-        )
-      )
+          child: new Text('Z'),
+        ),
+      ),
     );
 
     final RenderBox box = tester.renderObject(find.byType(CircleAvatar));
```

Comment:
if you replace RenderBox here with the actual type of the render object you're expecting, the `box.child` line below will work in the analyzer.